Marlin/Conditionals_post.h

```diff
@@ -306,13 +306,13 @@
   /**
    * ARRAY_BY_EXTRUDERS based on EXTRUDERS
    */
-  #define ARRAY_BY_EXTRUDERS(args...) ARRAY_N(EXTRUDERS, args)
+  #define ARRAY_BY_EXTRUDERS(...) ARRAY_N(EXTRUDERS, __VA_ARGS__)
```

Comment:
I like `__VA_ARGS__` if this is the better way to go. What makes it superior to using the named `args...` parameter?